ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1724,6 +1724,8 @@ private class JdbcLedgerDao(
         |truncate participant_command_completions cascade;
         |truncate participant_command_submissions cascade;
         |truncate participant_events cascade;
+        |truncate participant_event_flat_transaction_witnesses cascade;
```

Comment:
shouldn't this be taken care of by `truncate participant_events cascade;`?

Reply:
I'd say yes, but I'd say I'll add a test right now to make sure that's the case (and to ensure we don't forget about this in the future).